### ISSUE TITLE
Global Settings or Settings Template

### ISSUE THREAD

**Issue by tim-elmer** [ModOrganizer2/modorganizer#1982]
**Is your feature request related to a problem? Please describe.**
I'm a dark mode enjoyer. Getting flashbanged when I create a new MO2 instance is a wee bit irritating.

In general, having to set certain options for each instance seems backwards.

**Describe the solution you'd like**
It'd be nice if we could set certain settings globally across instances. Things that would logically apply to more than one instance, and can be set across all instances, overwriting whatever their current or default value may be.

Basically everything except for **Paths** and **Workarounds** (and maybe **Profile Defaults**). For example, does it really make sense to have different application update settings per-instance?

**Describe alternatives you've considered**
Rather than globally modifying instances, adding a "template" or "default" instance whose settings are copied to new instances would also address this irritation.

**Additional context**
Not really.
